src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -45,20 +45,22 @@ static ucs_config_field_t uct_cuda_copy_md_config_table[] = {
 
 static ucs_status_t uct_cuda_copy_md_query(uct_md_h md, uct_md_attr_t *md_attr)
 {
-    md_attr->cap.flags            = UCT_MD_FLAG_REG | UCT_MD_FLAG_ALLOC;
-    md_attr->cap.reg_mem_types    = UCS_BIT(UCS_MEMORY_TYPE_HOST) |
-                                    UCS_BIT(UCS_MEMORY_TYPE_CUDA) |
-                                    UCS_BIT(UCS_MEMORY_TYPE_CUDA_MANAGED);
-    md_attr->cap.alloc_mem_types  = UCS_BIT(UCS_MEMORY_TYPE_CUDA) |
-                                    UCS_BIT(UCS_MEMORY_TYPE_CUDA_MANAGED);
-    md_attr->cap.access_mem_types = UCS_BIT(UCS_MEMORY_TYPE_CUDA) |
-                                    UCS_BIT(UCS_MEMORY_TYPE_CUDA_MANAGED);
-    md_attr->cap.detect_mem_types = UCS_BIT(UCS_MEMORY_TYPE_CUDA) |
-                                    UCS_BIT(UCS_MEMORY_TYPE_CUDA_MANAGED);
-    md_attr->cap.max_alloc        = SIZE_MAX;
-    md_attr->cap.max_reg          = ULONG_MAX;
-    md_attr->rkey_packed_size     = 0;
-    md_attr->reg_cost             = UCS_LINEAR_FUNC_ZERO;
+    md_attr->cap.flags               = UCT_MD_FLAG_REG | UCT_MD_FLAG_ALLOC;
```

Comment:
introduce uct_md_attr_init()

Reply:
@yosefe it seems that if we do it, it will require setting some fileds to initial value and then re-initialize it. is it ok?

Reply:
yes

Reply:
done